docs/developer/payments/transactions.mdx

```diff
@@ -412,6 +412,12 @@ Query variables:
 }
 ```
 
+:::info
+If any issues occur during checkout validation (e.g., checkout contains unavailable items), the checkout will not be completed.
```

Comment:
I would suggest adding the definition of `unavailable items`. Right now it could not be fully clear what exactly can cause this.